fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java

```diff
@@ -132,6 +132,9 @@ private StorageVolume getStorageVolumeOfDb(String svName) throws DdlException {
     // Because it has been checked when creating db.
     private boolean bindDbToStorageVolume(String svId, long dbId, boolean isReplay) {
         try (LockCloseable lock = new LockCloseable(rwLock.writeLock())) {
+            if (svId == null) {
```

Comment:
null check should out of the lock. reduce unnecessary lock as much as possible.

Reply:
done